cg/store/crud/read.py

```diff
@@ -1545,3 +1542,61 @@ def get_case_ids_for_samples(self, sample_ids: list[int]) -> list[str]:
         for sample_id in sample_ids:
             case_ids.extend(self.get_case_ids_with_sample(sample_id))
         return list(set(case_ids))
+
+    def get_related_dna_samples_from_sample_within_collaborators(
```

Comment:
get_related_samples is maybe more fitting

Reply:
My philosophy when naming functions is that the entire signature should be considered. Meaning that if you have a parameter called `collaborators` it can be implied that the method in this case filters on it, and therefore we don't not need to add it to the function name.

Reply:
Got it! Thanks for the explanation! Corrected!